### DISCUSSION TITLE
Add support for local-only tables

### DISCUSSION THREAD

**Discussion by balegas** [electric-sql/electric#572] [Feature requests]
Currently the only way of having a table on the client is through Electrification, which doesn't allow developers to specify tables that live only on the local database.

To have client-only tables the developer has to drop down to raw access without any of the guarantees that Electrified offers on top (type-safety, managed migrations, ...).

This feature is concerned with adding support for creating local-only tables using Electric

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
